nginx.conf

```diff
@@ -1,34 +1,64 @@
-server {
-    listen       80;
-    server_name  localhost;
+worker_processes  1;
 
-    location / {
-        root   /usr/share/nginx/html;
-        index  index.html;
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
 
-        # "Pretty" URL-s: don't require the .html file extension
-        try_files $uri.html $uri $uri/ $uri/index.html =404;
-        
-    }
 
-    error_page  404              /404.html;
+events {
+    worker_connections  1024;
+}
+
 
-    error_page   500 502 503 504  /50x.html;
-    location = /50x.html {
-        root   /usr/share/nginx/html;
-    }
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
 
-    # Add some custom headers
-    add_header Strict-Transport-Security "max-age=604800";
-    add_header X-Want-To-Work-With "Well designed HTTP API-s";
-    add_header X-Recruitment "https://linkedin.com/in/anroots";
-    add_header X-Powered-By "Jekyll";
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
 
-    # Hide the server version from the headers
-    server_tokens off;
+    access_log  /var/log/nginx/access.log  main;
 
-    # Only some select HTTP methods are allowed
-    if ($request_method !~ ^(GET|HEAD|OPTIONS)$ ) {
-        return 405;
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    gzip  on;
+    
+    server {
+        listen       8080;
```

Comment:
This starts the server on port `8080` instead of the default `80` as port `80` is a privileged port and requires root.